pkg/kubelet/dockertools/manager.go

```diff
@@ -588,11 +588,13 @@ func (dm *DockerManager) runContainer(
 			Hostname:     containerHostname,
 			Image:        container.Image,
 			// Memory and CPU are set here for older versions of Docker (pre-1.6).
-			Memory:     memoryLimit,
-			MemorySwap: -1,
-			CPUShares:  cpuShares,
-			WorkingDir: container.WorkingDir,
-			Labels:     labels,
+			Memory:      memoryLimit,
+			MemorySwap:  -1,
+			CPUShares:   cpuShares,
+			WorkingDir:  container.WorkingDir,
+			Labels:      labels,
+			AttachStdin: container.Stdin,
```

Comment:
I think it has to be best effort.  I'm happy to log an event or something to indicate the error.